pkg/scheduler/plugins/elasticquota/plugin_helper.go

```diff
@@ -59,6 +61,17 @@ var GetQuotaName = func(pod *v1.Pod, quotaLister schedulinglisterv1alpha1.Elasti
 	} else if !errors.IsNotFound(err) {
 		klog.Errorf("Failed to Get ElasticQuota %s, err: %v", pod.Namespace, err)
 	}
+
+	eqList, err := quotaLister.List(labels.Everything())
```

Comment:
The operation overhead of List is a bit high, especially when there are a lot of quotas, there will be a lot of unnecessary overhead, and the performance will be poor, so should we do indexing?